packages/engine/tests/engine/input/systems/ButtonCleanupSystems.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import assert from 'assert'
+
+import { destroyEngine } from '../../../../src/ecs/classes/Engine'
+import { defineQuery } from '../../../../src/ecs/functions/ComponentFunctions'
+import { SystemDefinitions } from '../../../../src/ecs/functions/SystemFunctions'
+import { createEngine } from '../../../../src/initializeEngine'
+import { ButtonCleanupSystem } from '../../../../src/input/systems/ButtonCleanupSystem'
+import { loadEmptyScene } from '../../../util/loadEmptyScene'
+
+describe('ButtonCleanupSystem', () => {
+  beforeEach(() => {
+    createEngine()
+    loadEmptyScene()
+  })
+
+  it('should be queryable', () => {
+    const buttonCleanupQuery = defineQuery([ButtonCleanupSystem])
+    assert(buttonCleanupQuery)
+    assert(buttonCleanupQuery._enterQuery)
+    assert(buttonCleanupQuery._exitQuery)
+    assert(buttonCleanupQuery._query)
+  })
+
+  it('test button cleanup system', () => {
+    const system = SystemDefinitions.get(ButtonCleanupSystem)!
+    const execute = system.execute
+    assert(typeof execute === 'function')
+    assert(execute() === undefined)
```

Comment:
we don't need to assert the return type of execute, nor the type of the property, just the functionality